daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -31,7 +31,9 @@ The table below shows the versions of Dapr releases that have been tested togeth
 
 | Release date | Runtime     | CLI  | SDKs  | Dashboard  | Status |
 |--------------------|:--------:|:--------|---------|---------|---------|
-| Feb 17th 2021 | 1.0.1</br>(Mar 4th 2021) | 1.0.0 | Java 1.0.0 </br>Go 1.0.0 </br>PHP 1.0.0 </br>Python 1.0.0 </br>.NET 1.0.0 | 0.6.0 | Supported (current) |
+| Feb 17th 2021 | 1.0.0</br>| 1.0.0 | Java 1.0.0 </br>Go 1.0.0 </br>PHP 1.0.0 </br>Python 1.0.0 </br>.NET 1.0.0 | 0.6.0 | Supported |
+| Mar 4th 2021  | 1.0.1</br>| 1.0.1 | Java 1.0.0 </br>Go 1.0.0 </br>PHP 1.0.0 </br>Python 1.0.0 </br>.NET 1.0.0 | 0.6.0 | Supported |
+| Mar 30th 2021 | 1.1.0</br> | 1.1.0 | Java 1.1.0 </br>Go 1.1.0 </br>PHP 1.1.0 </br>Python 1.1.0 </br>.NET 1.1.0 | 0.6.0 | Supported (current) |
```

Comment:
Java it is still 1.0.2 ... not 1.1.0 ...

Reply:
Thanks, I wasn't sure about the SDK version. 

@wcs1only @rynowak @skyao @withinboredom can you provide the correct versions for the Python, .NET, Go and PHP SDKs that work with v1.1 respectively?

Reply:
new .NET SDK release will be 1.1.0, however 1.0.0 should still be working